frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -791,6 +791,93 @@ describe("Where's My Gene", () => {
       }
     });
   });
+
+  describe("Newsletter", () => {
+    const NEWSLETTER_MODAL_CONTENT = "newsletter-modal-content";
+    const NEWSLETTER_MODAL_OPEN_BUTTON = "newsletter-modal-open-button";
+    const NEWSLETTER_MODAL_CLOSE_BUTTON = "newsletter-modal-close-button";
+    const NEWSLETTER_SUBSCRIBE_BUTTON = "newsletter-subscribe-button";
+    const NEWSLETTER_EMAIL_INPUT = "newsletter-email-input";
+    const NEWSLETTER_VALIDATION_ERROR_MESSAGE =
+      "newsletter-validation-error-message";
+    const FAILED_EMAIL_VALIDATION_STRING =
+      "Please provide a valid email address.";
+
+    test("Newsletter Modal - Open/Close", async ({ page }) => {
+      await goToPage(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`, page);
+
+      // Open modal
+      await getButtonAndClick(page, NEWSLETTER_MODAL_OPEN_BUTTON);
+
+      await waitForElement(page, NEWSLETTER_MODAL_CONTENT);
+
+      const modalContent = page.getByTestId(NEWSLETTER_MODAL_CONTENT);
+
+      // modal content
+      expect(modalContent.getByText("Join Our Newsletter")).toBeTruthy();
+      expect(
+        modalContent.getByText(
+          "Get a quarterly email with the latest CELLxGENE features and data."
+        )
+      ).toBeTruthy();
+      expect(modalContent.getByText("Enter email address")).toBeTruthy();
+      expect(modalContent.getByText("Subscribe")).toBeTruthy();
+      expect(modalContent.getByText("Unsubscribe at any time.")).toBeTruthy();
+
+      // Close modal
+      await getButtonAndClick(page, NEWSLETTER_MODAL_CLOSE_BUTTON);
+
+      await waitForElementToBeRemoved(page, NEWSLETTER_MODAL_CONTENT);
+    });
+
+    test("Newsletter Modal - Validate Email", async ({ page }) => {
+      await goToPage(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`, page);
```

Comment:
Is this missing a couple of tests?

1. valid email format. ex: `dbrooks@grizzlies.com`
2. valid email format but email does not exist. Maybe you get one of these SMTP errors in the backend that shows up something else on the frontend? https://support.google.com/a/answer/3221692?hl=en
3. Lastly a valid email address that does exist.